src/validator.cpp

```diff
@@ -1929,7 +1929,7 @@ void Validator::ValidatorImpl::validateMathMLElementsChildrenAndSiblings(const X
             }
         }
 
-        // Assignment, and relational and logical operators.
+        // Equality, and relational and logical operators.
```

Comment:
probably don't need the `Equality` here - these are just relational and logical operators.

Reply:
fee74ad9